test/barrel_task_status_tests.erl

```diff
@@ -16,7 +16,7 @@
 -module(barrel_task_status_tests).
 
 -include_lib("eunit/include/eunit.hrl").
--include("db.hrl").
+-include("../include/db.hrl").
```

Comment:
When tests are run via mix eunit or ct, tests are not able to access include folder directly, maybe need to set erl options.